docs/database/cosmosdb.md

```diff
@@ -440,5 +448,743 @@ It captures the number of active instances at any given time. Best practices dic
+| [`db.cosmosdb.network.host`](/docs/attributes-registry/db.md) | string | Host from Gateway or Backend endpoint address | `rntbd://cdb-east-us.azure.document.com` | `Conditionally Required` If available | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| [`db.cosmosdb.network.port`](/docs/attributes-registry/db.md) | int | Port from Gateway or Backend endpoint address | `443` | `Conditionally Required` If available | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| [`db.cosmosdb.network.region`](/docs/attributes-registry/db.md) | string | Azure Cosmos DB Region where the network call was made | `WEST US 2` | `Conditionally Required` If available | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| [`db.cosmosdb.network.response.status_code`](/docs/attributes-registry/db.md) | int | HTTP Status code returned for network call | `200` | `Conditionally Required` If available | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| [`db.cosmosdb.network.response.sub_status_code`](/docs/attributes-registry/db.md) | int | Azure Cosmos DB Sub Status code returned for network call | `1002` | `Conditionally Required` If available | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| [`db.cosmosdb.sub_status_code`](/docs/attributes-registry/db.md) | int | Cosmos DB sub status code. | `1000`; `1002` | `Conditionally Required` when response was received and contained sub-code. | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| [`db.namespace`](/docs/attributes-registry/db.md) | string | The name of the database, fully qualified within the server address and port. | `customers`; `test.users` | `Conditionally Required` If available. | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| [`db.operation.name`](/docs/attributes-registry/db.md) | string | The name of the operation or command being executed. [2] | `findAndModify`; `HMSET`; `SELECT` | `Conditionally Required` [3] | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| [`db.response.status_code`](/docs/attributes-registry/db.md) | string | Database response status code. [4] | `102`; `ORA-17002`; `08P01`; `404` | `Conditionally Required` [5] | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| [`error.type`](/docs/attributes-registry/error.md) | string | Describes a class of error the operation ended with. [6] | `timeout`; `java.net.UnknownHostException`; `server_certificate_invalid`; `500` | `Conditionally Required` If and only if the operation failed. | ![Stable](https://img.shields.io/badge/-stable-lightgreen) |
```

Comment:
I am a bit confused - pretty much all exceptions thrown form the SDK to the application would be a CosmosException. Maybe one of teh sub-classes in Java. The sample here should never be thrown to applicatin - unless the intent is to use not the exception thrown to app but the inner most inner exception? Which is the desired exception to be used?

Reply:
Put the appropriate examples.